target/linux/mediatek/image/filogic.mk

```diff
@@ -105,10 +105,25 @@ define Build/cetron-header
 	rm $@.tmp
 endef
 
-define Device/asus_tuf-ax4200
+define Device/acer_predator-w6
```

Comment:
Please add a seperate commit before this one which moves the `acer_predator-w6` device to the position you moved it to to make it easier to review your changes.

Reply:
done!